setup.py

```diff
@@ -19,6 +19,7 @@
 except ImportError:
     cythonize = None
 
+sys.path += [os.path.dirname(__file__)]
```

Comment:
iirc this was also related to the pyproject.toml / black change. so do not backport in here.